components/notion/notion.app.mjs

```diff
@@ -325,7 +325,8 @@ export default {
         block_id: blockId,
       });
     },
-    async retrieveBlockChildren(block, params = {}) {
+    async retrieveBlockChildren(block, subpagesOnly = false) {
```

Comment:
For the record, I've checked that all instances of this method being called did not include a `params` argument, so it was always initialized as an empty object - hence I changed it to a variable declaration instead.